tensorflow/core/platform/default/build_config.bzl

```diff
@@ -638,7 +638,9 @@ def tf_additional_core_deps():
         clean_dep("//tensorflow:android"): [],
         clean_dep("//tensorflow:ios"): [],
         clean_dep("//tensorflow:linux_s390x"): [],
-        clean_dep("//tensorflow:windows"): [],
+        clean_dep("//tensorflow:windows"): [
```

Comment:
can't you just remove the "//tensorflow:windows" entry and it'll fall through to default?